src/shared/components/annotation/OncoKB.tsx

```diff
@@ -78,7 +78,15 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
             <span className={`${annotationStyles["annotation-item"]}`} />
         );
 
-        if (this.props.indicator)
+        if (this.props.indicator === null) {
+            // null means there is an error...
+            oncoKbContent = this.errorIcon();
```

Comment:
Probably, this is not the best way to distinguish between `error` and `loading` states. Ideally there should be a `status` indicator for each remote data. But currently, we have the `status` indicator only for `cache` data.